src/Controller/Api/UserController.ts

```diff
@@ -53,10 +53,10 @@ export class UserController {
 
     public async getAllUserNodes(req: Request, res: Response) {
```

Comment:
I just realized this should be in node controller because we are fetching nodes. Same will be for disk info.

Therefore, routes should be `user/node` and `user/diskinfo` etc.

Reply:
I'll move it to the node controller.